packages/react-native/src/modules/feed/components/NotificationFeedContainer/NotificationFeedContainer.tsx

```diff
@@ -1,7 +1,7 @@
 import React from "react";
 import { View } from "react-native";
 
-export const KnockFeedContainer: React.FC<React.PropsWithChildren> = ({
+export const NotificationFeedContainer: React.FC<React.PropsWithChildren> = ({
```

Comment:
Not apart of the changed code but curious why `React.FC<React.PropsWithChildren>` is here vs defining the prop types directly to the props?

Reply:
Do you mean why use `React.PropsWithChildren` instead of creating a props type with a child property? There's not a huge difference but I think using `PropsWithChildren` is more consistent (e.g. if React changes the type children should be).

Reply:
Good idea! Yeah, at this point this component is mainly to test that the build is working as expected. In the web react version of this package, the `NotificationFeedContainer` components imports some base styles which we don't have yet. I think for now having it as a plain wrapper is okay.